src/Analyzers/CSharp/Analyzers/InvokeDelegateWithConditionalAccess/InvokeDelegateWithConditionalAccessAnalyzer.cs

```diff
@@ -87,6 +87,12 @@ private void SyntaxNodeAction(SyntaxNodeAnalysisContext syntaxContext)
                 return;
             }
 
+            // Function pointers can only be invoked directly via the "()" operator, not "?.Invoke()".
```

Comment:
@333fred i'm curious why that is.  is there no `check for null and invoke` idiom for function pointers?